services/director-v2/src/simcore_service_director_v2/utils/dags.py

```diff
@@ -69,12 +71,19 @@ async def compute_node_modified_state(
     nodes_data_view: nx.classes.reportviews.NodeDataView, node_id: NodeID
 ) -> bool:
     node = nodes_data_view[str(node_id)]
+    # if the node state is in the modified state already
+    if not node["state"] or node["state"] in [
```

Comment:
can ``bool(node['state']) == False`` e.g. ``0, None, False, ''`` etc ??
would it be more clear as
```python
if node['state'] in (None, RunningState.ABORTED, RunningStateFAILED):
    return True
```